src/main/java/Environment/Metrics/S_Metrics.java

```diff
@@ -177,6 +178,31 @@ public static void addOutputStream(OutputStream outputStream) throws IOException
         addOutputStream(outputStream, S_Metrics::instanceReportToStringCSV, S_Metrics::headerArrayToStringCSV);
     }
 
+    public static void addSolutionExportOutputStream(String pathName) throws IOException {
+        int index = S_Metrics.getAllReports().size() - 1;
+
+        if (verifyOutputPath(pathName)) {
+            String fullPathName = pathName + "/" + index + ".txt";
+            addOutputStream(new FileOutputStream(fullPathName), S_Metrics::instanceReportToSolutionString);
+        }
+    }
+
+
+    public static boolean verifyOutputPath(String path) {
+        File directory = new File(path);
+        if (! directory.exists()){
+            boolean created = directory.mkdir();
+            if(!created){
+                String errString = "Could not locate or create output directory.";
+                System.out.println(errString);
+                return false;
+            }
+        }
+        return true;
+    }
+
+
+
```

Comment:
Why implement it again? you're already importing it from A_RunManager. In fact, it's probably better to move it to IO_Manager and have both import it from there.